openassessment/data.py

```diff
@@ -993,15 +997,26 @@ def create_zip_with_attachments(cls, file, course_id, submission_files_data):
 
         with ZipFile(file, 'w') as zip_file:
             for file_data in submission_files_data:
-                file_content = (
-                    cls._download_file_by_key(file_data['key'])
-                    if file_data['type'] == cls.ATTACHMENT
-                    else file_data['content']
-                )
+                key = file_data['key']
+                file_name = file_data['file_path']
```

Comment:
nit: `file_name` -> `file_path` just for consistency's sake, plus, technically I think it is actually a file path